pkg/cmd/workspace/create.go

```diff
@@ -407,3 +411,11 @@ func getEnvVariables(project *apiclient.CreateWorkspaceRequestProject, profileDa
 
 	return &envVars
 }
+
+func resetTerminal() error {
+	cmd := exec.Command("reset")
```

Comment:
Does this work on all OSs?

I believe the solution to the issue should be to write a terminal character that shows the cursor and that's it.

Reply:
this function uses the reset command, which is available on most systems like
![image](https://github.com/user-attachments/assets/ee3f6c57-f7d8-4f86-b20c-aae14b5f7fa2)

> I believe the solution to the issue should be to write a terminal character that shows the cursor and that's it.
Made a new commit. That's the easiest one I guess. I make things complex.